examples/torch/object_detection/layers/modules/ssd_head.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, outputs, modules):
         super().__init__(*modules)
         self.outputs = [str(o) for o in outputs]
 
+    #pylint:disable=W0237
```

Comment:
Looks like this warning happens when a method parameter has a different name than in the implemented interface or in an overridden method.
Can we just simply rename the parameter?

Reply:
This parameter was originally named input and if rename it, then there will be the warning W0622: Redefining built-in 'input'

Reply:
ok, let's keep the original W0237 warning